_episodes/02-setup.md

```diff
@@ -101,15 +101,23 @@ your changes, press <kbd>Esc</kbd> then type `:q!` and hit <kbd>Return</kbd>.
 The four commands we just ran above only need to be run once: the flag `--global` tells Git
 to use the settings for every project, in your user account, on this computer.
 
-You can check your settings at any time:
+Let's review those settings and test our `core.editor` right away:
+
+~~~
+$ git config --global --edit
+~~~
+{: .language-bash}
+
+However, it's safer to only view the configuration with:
```

Comment:
How about
> However, since typos in the config file will cause issues, it's safer to only view the configuration with:

?

Reply:
Hi @kekoziar 👋 You commented on an older version of the diff. Is the current one also in need of changes? If so, please note that I've activated `Allow edits by maintainers`. Please feel free to apply your favourite wording directly. Cheers :-)